core/baseline-core-v1.0-psd01.md

```diff
@@ -417,7 +414,8 @@ A counterparty that provides one or more State Object to another counterparty is
 
 A Requester may request State Objects from multiple Providers, and in the context of a supply chain of State Objects, a Provider of one State Object may also play the role of a Requester of other State Objects.
 
-**[R1]**	Transacting counterparties  MUST have a corresponding agreement.
+#### **[R1]**
+Transacting counterparties  MUST have a corresponding agreement.
```

Comment:
Extra whitespace here.

Can address minor things like this on a final pass methinks.